modules/ROOT/pages/create-instance-task-flex.adoc

```diff
@@ -24,18 +24,141 @@ See xref:gateway::flex-install.adoc[Install Flex Gateway] and xref:gateway::flex
+
+image:multiple-upstreams.png[Flex Gateway manages the traffic to multiple upstreams]
+
+=== Routes
+
+Each API instance supports up to 10 routes and each route can support up to 10 upstream services. You configure what routes receive what requests by defining route rules and route order. Only one route per API instance is required. 
+
+You can add additional routes by clicking *Add Route (optional)*, and you can delete routes by clicking the *Trash Can* icon (image:trash-can-icon.png[2%,2%]). If only one route is defined, you cannot delete that route.
+
+==== Route Rules
+
+You can direct requests to different routes using route rules. 
+
+To view and edit route rules, click *Route Rules*.
+
+All rules are optional. Not defining a value for a rule means that aspect of the request not considered. For example, not specifying a host means that the route can service any host given the request meets the other route rules. Not defining any rules means that the route can service every request.
+
+Additionally, different routes can support the same upstream services. If you cannot capture all the requests for a particular set of upstream services in a single route rule set, you can define multiple routes with different rules for full coverage.
+
+Only requests that meet all rules defined in the ruleset are directed to that route.
+
+Flex Gateway supports the following route rules:
+
+Method::::
++
+The *Method* rule defines the types of request methods the route can service.
++
+You can select multiple methods for each route. Only requests that are one of the defined methods are directed to this route.
++
+To select the supported methods:
++
+. Expand the *Method* drop down list.
+. Select the supported methods.
++
+Path::::
++
+The *Path* rule defines the request path the route can service.
++
+You can only define one “URI Template Regex" path for each route. Only requests with the defined path are sent to this route.
++
+To define the *Path* rule:
++
+. Enter your path template in the *Path* configuration field.
++
+Host::::
++
+The *Host* rule defines the request host the route can service.
++
+You can only define one host URL for each route. Only requests made from the defined host are sent to this route.
++
+To define the *Host* rule:
++
+. Enter your host in the *Host* configuration field.
++
+Header::::
++
+The *Headers* rule defines what headers are required to be present or not present for this route to service the request.
++
+For this rule, you must define the header name and a `true` or `false` value of whether or not the header is present. Only requests that meet all the specified header requirements are sent to this route. Additional headers present in the request that are not specifically defined in the rules are ignored. 
```

Comment:
Are they writing `true` and `false` or selecting this value?

Reply:
Writing